src/main/java/uk/gov/ch/developer/docs/interceptor/LoggingInterceptor.java

```diff
@@ -15,12 +15,11 @@
 @Component
 public class LoggingInterceptor extends HandlerInterceptorAdapter {
 
+    private final Logger logger;
     private RequestLogger requestLogger = new RequestLogger() {
         // Implement Defaults
```

Comment:
Delete comment

Reply:
I disagree. I think this is worth putting here to say that we are deliberately not implementing or overriding methods.

Reply:
Why would that be of any benefit? As a default non-descriptive comments should be removed or expanded on.